### ISSUE TITLE
New/edited Custom Data Viewer Layouts don't appear as choices

### ISSUE THREAD

**Issue by tatiang** [thunkable/thunkable-issues#1125]
### Platform
Drag and Drop UI - web

### Steps to Reproduce
1. Create a new screen. 
2. Add a button within a row within a row. 
3. Give the button a background color.
4. Give the rows fit contents for height and fill container for width.
5. Save the outer row as a Data Viewer Layout with grid selected.

### Expected Behavior
The newly saved layout should appear when selecting a Data Viewer Grid layout in another project. Over the past day, this has occurred about 10% of the time when I save/edit a custom layout.

### Actual Behavior
The newly saved layout does not appear when selecting a Data Viewer Grid layout in another project.

### Project Link
Project with custom layout on Screen3: https://x.thunkable.com/copy/f29b1c29601a185786700b629d7a6be2

###Community Link

```
  STACK TRACE AND/OR SCREENSHOTS
```

**Comment by tatiang** [comment]
I noticed that when I add a Data Viewer List, I have all of the choices that I saved as "grid" type layouts. But when I add a Data Viewer Grid, I don't have those choices. It appears that almost everything I save as a "grid" is actually getting saved as a List layout.

**Comment by jaredgibb** [comment]
I was unable to reproduce this. Can you see this video. 

this might take further debugging. 

https://vimeo.com/682866256

**Comment by tatiang** [comment]
> Can you see this video.

Yes but the video is just 1 minute of nothing, essentially a screenshot. Did you mean to upload that?

**Comment by tatiang** [comment]
Here's a video (no sound) of the problem I'm seeing:

https://watch.screencastify.com/v/pu1kNblen9NjV5h0zbK7

In the video, I change the color of a layout component and save it as a custom "grid" layout. I then attempt to select it for a Data Viewer Grid (DVG) and it works. I repeat this process but this time, it's missing from the list of choices. 

I then unhide a Data Viewer List (DVL) and attempt to select that second layout from the list of choices. It's there even though it's a "grid" layout.

**Comment by jaredgibb** [comment]
That's strange. Ok, thank you for flagging this. Would you mind testing a refresh after each time you create a new DVL as a temporary workaround. I'll flag this as a bug report. 

For now, the above workaround should work.

**Comment by tatiang** [comment]
What do you mean by a refresh? I've tried closing the project and re-opening it but that didn't seem to make a difference.

And what workaround? I didn't find a consistent method to get this to work. Because if I create it as a grid and assign it to a DVL, that defeats the purpose if I need the layout as a DVG. I think.

**Comment by jaredgibb** [comment]
For me, when i refreshed the screen and saved the modified layout, there was no issue. 

https://www.loom.com/share/13822e5b66294240a7b7094f5cd2d43e

**Comment by tatiang** [comment]
I'll try refreshing and saving. Thank you.
